src/components/auth/login/GoogleAndFacebookLogIn.jsx

```diff
@@ -39,27 +37,16 @@ export default function GoogleAndFacebookLogIn() {
   };
 
   return (
-    <Row className="socialLoginButtons">
-      <Col>
-        <Button
-          variant="info"
-          size="md"
-          className="fbButton"
-          onClick={() => logIn(facebookProvider)}
-        >
-          Facebook
-        </Button>
-      </Col>
-      <Col>
-        <Button
-          variant="info"
-          size="md"
-          className="googleButton"
-          onClick={() => logIn(googleProvider)}
-        >
-          Google
-        </Button>
-      </Col>
+    <Row className="socialLoginIcons">
+      <span>Login or Register using:</span>
+      <i
```

Comment:
why did you use a <Row> without a <Col> ?

Reply:
What do you mean? I didn't understand.

Reply:
Col could be used as well but didnt want to add too many unnecessarty wrapper.